### PR TITLE
Translated readme.

### PR DESCRIPTION
**Pull request by davidhsrtoyota**
Translated readme.
Verification, please.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
 - **urdf**
 
-    共通マクロを含んだxacroファイルが直下にある。
+    The xacro files that include common macros are directly under this directory.
 
-    各パーツは `${部品名}_${バージョン}` という名前のフォルダに別れて配置されている。
+    Each part is separated into folders named ${PARTNAME}_${VERSION}.
 
-    ハードウェアの改修によって大きなモデル修正が起きた場合は、バージョンを1つ上げて新しくフォルダを設けること。
+    When a big model revision is caused by hardware repairs, the version is increased by one and a new folder is created.
```
Comment (yuka-hashiguchi) [inline README.rst:29]: how about "major change such as model revision" instead of "big model revision"?

**Review by yuka-hashiguchi — COMMENTED** [review]
Please check my two comments.

File: README.rst [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
 
-本パッケージはHSR-Bのロボットモデルを管理する。
-Gazeboシミュレータ用設定も含む。
+This package manages the HSR-B robot model.
+Gazebo simulator configurations are also included.。
 
 
-推奨動作環境
-++++++++++++
+Recommended environment
```
[the remark this answers is not in the document]
Comment (davidhsrtoyota) [reply README.rst:8]: No problem.  I will delete it.

File: README.rst (hunk 1 above)
Comment (davidhsrtoyota) [reply README.rst:29]: Yes.  Good word choice. :)
I will make the edit.

**Review by yuka-hashiguchi — APPROVED** [review]
Thank you for all your support, David!

**Pull request merged**